cli/wallet/wallet.go

```diff
@@ -20,13 +20,13 @@ import (
 )
 
 var (
-	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--path, -p) flags")
+	errNoPath         = errors.New("target path where the wallet should be stored is mandatory and should be passed using (--path, -w) flags")
 	errPhraseMismatch = errors.New("the entered pass-phrases do not match. Maybe you have misspelled them")
 )
 
 var (
 	walletPathFlag = cli.StringFlag{
-		Name:  "path, p",
+		Name:  "path, w",
```

Comment:
`wallet, w`? And we have `-p` mentioned in `docs`, so we need to update them too.

Reply:
Fixed.